README.md

```diff
@@ -1,2 +1,44 @@
-# bookstore
-Bookstore is an MVP version that allows the user to store and display books
```

Comment:
- [ ] Please kindly update your readme file to reflect the changes in this milestone as this will help users and potential recruiters to quickly get an idea of what you project is about. Please follow this [template](https://github.com/microverseinc/readme-template) and adapt it to your project.